package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@financial-times/sass",
+  "version": "1.26.2",
```

Comment:
This tracks the version of dart-sass

Reply:
is this a good idea? what if we need to release a patch of our own to `@financial-times/sass`?

Reply:
I've changed it to use whatever version we choose